fdbserver/StorageMetrics.actor.h

```diff
@@ -266,6 +271,23 @@ struct StorageServerMetrics {
 		}
 	}
 
+	// Due to the fact that read sampling will be called on all reads, use this specialized function to avoid overhead
+	// around branch misses and unnecessary stack allocation which eventually addes up under heavy load.
+	void notifyBytesReadPerKSecond(KeyRef key, int64_t in) {
```

Comment:
Since this function mimics the `notify()`, it will be easier to correlate these two function if you could move `notifyBytesReadPerKSecond ` immediately after `notify()`.

Reply:
Done.